inst/rmarkdown/resources/html/header_polyfill.html

```diff
@@ -0,0 +1,13 @@
+<style>
+  @media print {
+    .btn-print {
+      display: none;
+    }
+  }
+</style>
+
+<script>
+  PagedConfig = {auto: false};
```

Comment:
Plus explicite et plus safe

```suggestion
  window.PagedConfig = {auto: false};
```